@here/harp-examples/src/hello.ts

```diff
@@ -124,4 +125,26 @@ export namespace HelloWorldExample {
     // snippet:harp_gl_hello_world_example_5.ts
     mapView.addDataSource(omvDataSource);
     // end:harp_gl_hello_world_example_5.ts
+
+    const gui = new GUI({ width: 300 });
+    const options = {
+        theme: {
+            day: "resources/berlin_tilezen_base.json",
+            reducedDay: "resources/berlin_tilezen_day_reduced.json",
+            reducedNight: "resources/berlin_tilezen_night_reduced.json",
+            streets: "resources/berlin_tilezen_effects_streets.json",
+            outlines: "resources/berlin_tilezen_effects_outlines.json",
+            textured: "resources/berlin_tilezen_textured.json"
+        }
+    };
+    gui.add(options, "theme", options.theme).onChange((value: string) => {
```

Comment:
The dropdown list is initially empty. Find how to display the initial theme name as the active one in the dropdown list.

Reply:
Also add the new themes in mapview-demo

Reply:
not possible.